lib/Driver/Driver.cpp

```diff
@@ -2009,6 +2010,7 @@ void Driver::printHelp(bool ShowHidden) const {
     break;
   case DriverKind::Batch:
   case DriverKind::AutolinkExtract:
+  case DriverKind::SwiftFormat:
     ExcludedFlagsBitmask |= options::NoBatchOption;
```

Comment:
It may be interesting to have this available as a batch mode operation, where you could format using the same invocation as a compile.